src/Shared/ReadOnlyEmptyDictionary.cs

```diff
@@ -256,7 +256,8 @@ public bool Remove(KeyValuePair<K, V> item)
         /// </summary>
         public IEnumerator<KeyValuePair<K, V>> GetEnumerator()
         {
-            return Enumerable.Empty<KeyValuePair<K, V>>().GetEnumerator();
+            IEnumerable<KeyValuePair<K, V>> enumerable = [];
+            return enumerable.GetEnumerator();
```

Comment:
nonblocking nit: I prefer the old way for this too; clearer to see that it should be a singleton.